sdk/src/test/java/com/horizen/account/transaction/EthereumTransactionSemanticValidityTest.java

```diff
@@ -407,4 +407,15 @@ public void testContractDeploymentValidity() {
         );
     }
 
+    @Test
+    public void testBigTxValidity() {
+        // data are charged with gas therefore we must set a gasLimt large enough.
+        // As of now check on block max gas limit is not made in tx.semanticValidity (it is made in state.validate(tx))
+        var goodTx = getBigDataTransaction(5000000, BigInteger.valueOf(100000000));
+        try {
+            goodTx.semanticValidity();
+        } catch (Throwable e) {
+            fail("Test1: Successful EthereumTransaction creation expected." + e);
```

Comment:
Consider to avoid catching the exception, because if the exception is caught by the test framework also the stack trace is printed, while in this case you print just the exception message

Reply:
Done